features/interview/service.py

```
@@ -139,13 +149,177 @@ def get_session_state(self, session_id: str) -> InterviewState | None:
+                    },
+                    ensure_ascii=False,
+                ),
+            }
+
+        except Exception as e:
+            logger.exception("SSE 스트리밍 중 예외 발생")
+            final_state = await self.get_session_state(session_id)
+            if final_state is None:
+                yield self._build_stream_error_event(
+                    code="final_state_missing",
+                    message=f"최종 상태를 가져오지 못했습니다. ({str(e)})",
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)